src/styles/index.scss

```diff
@@ -1,3 +1,52 @@
+@import '/src/styles/variables';
+
 body {
+  @include center-all;
+
   margin: 0;
+  height: 100vh;
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  width: calc((7 * $field-side) + (6 * $calendar-gap));
+  gap: $calendar-gap;
+
+  @each $start-day, $count in $days {
+    &--start-day-#{$start-day} :first-child {
+      margin-left: calc(($field-side + $calendar-gap) * $count);
+    }
+  }
+
+  @for $i from 28 through 31 {
+    &--month-length-#{$i} :nth-child(n + #{$i + 1}) {
+      display: none;
+    }
+  }
+}
+
+.calendar__field {
+  @include center-all;
+
+  height: $field-side;
+  width: $field-side;
```

Comment:
It's a good practice to combine properties that set the same values. In this case, you can use the `size` property instead of `height` and `width` to set both dimensions at once. It would look like this: `size: $field-side;`